src/gps/gpsTask.c

```diff
@@ -29,18 +29,21 @@ void GPSTask(void *pvParameters)
     Serial *serial = get_serial(SERIAL_GPS);
     uint8_t targetSampleRate = decodeSampleRate(getWorkingLoggerConfig()->GPSConfigs.speed.sampleRate);
     lapStats_init();
-    gps_status_t gps_status = GPS_init(targetSampleRate, serial);
-    if (!gps_status) {
-        pr_error("GPS: Error provisioning\r\n");
-    }
+    while(1) {
+        gps_status_t gps_status = GPS_init(targetSampleRate, serial);
```

Comment:
Maybe you want to have a log statement here saying you are provisioning?  Maybe its useless.  Thoughts?